### PR TITLE
modify padding idx

### PR DESCRIPTION
**Pull request by ken2403**

### CODE REVIEW

File: lcaonet/model/lcaonet.py [hunk 1]
```diff
@@ -686,7 +686,7 @@ def __init__(self, embed_dim: int, max_z: int = 36, max_orb: str | None = None,
 
         self.embed_dim = embed_dim
         self.extend_orb = extend_orb
-        self.e_embeds = nn.ModuleList([nn.Embedding(m, embed_dim) for m in MAX_IDX_BIG[: self.n_orb]])
+        self.e_embeds = nn.ModuleList([nn.Embedding(m, embed_dim, padding_idx=0) for m in MAX_IDX_BIG[: self.n_orb]])
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:689]: use torch `padding_idx` param
